src/style.css

```diff
@@ -1,3 +1,62 @@
 body {
   margin: 0;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  text-transform: uppercase;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+}
+
+.nav_link {
+  line-height: 15px;
+  text-align: center;
+  font-weight: 500;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+}
+
+img {
+  position: absolute;
```

Comment:
position absolute seems redundant here.

Reply:
Not fixed 
<img width="742" alt="image" src="https://github.com/user-attachments/assets/0bacdc7d-be75-4224-9158-54583897eb2d">